src/main/java/org/eclipse/cargotracker/interfaces/handling/rest/HandlingReport.java

```diff
@@ -8,35 +8,23 @@
 @XmlRootElement
 public class HandlingReport {
 
-  // TODO [Jakarta EE 8] Use not blank instead.
-  @NotNull(message = "Missing completion time.")
-  @Size(min = 16, max = 16, message = "Completion time value must be sixteen characters long.")
-  // TODO [DDD] Apply regular expression validation.
+  @NotNull
+  @Size(min = 16, max = 16)
```

Comment:
Could you kindly put back the error messages? I think they got removed accidentally?